From 3648b351e88331f3f2e46af4ae0ddd19e27bc7c2 Mon Sep 17 00:00:00 2001
From: Jakob Unterwurzacher <jakobunt@gmail.com>
Date: Mon, 8 Jul 2019 21:07:04 +0200
Subject: [PATCH] tests: fix TestCli failure when all processes have badness 0

On a machine with a lot of RAM and no big processes running,
all processes may have badness 0, and we will get

	^ new victim (higher vm_rss)

in the debug output, but not

	^ new victim (higher badness)

. This is not a failure.

Fixes https://github.com/rfjakob/earlyoom/issues/137
---
 tests/cli_test.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/cli_test.go b/tests/cli_test.go
index 7085290..ef4abee 100644
--- a/tests/cli_test.go
+++ b/tests/cli_test.go
@@ -72,7 +72,7 @@ func TestCli(t *testing.T) {
 		{args: nil, code: -1, stderrContains: startupMsg, stdoutContains: memReport},
 		{args: []string{"-p"}, code: -1, stdoutContains: memReport},
 		{args: []string{"-v"}, code: 0, stderrContains: "earlyoom v", stdoutEmpty: true},
-		{args: []string{"-d"}, code: -1, stdoutContains: "^ new victim (higher badness)"},
+		{args: []string{"-d"}, code: -1, stdoutContains: "^ new victim"},
 		{args: []string{"-m", "1"}, code: -1, stderrContains: " 1 %", stdoutContains: memReport},
 		{args: []string{"-m", "0"}, code: 15, stderrContains: "fatal", stdoutEmpty: true},
 		{args: []string{"-m", "-10"}, code: 15, stderrContains: "fatal", stdoutEmpty: true},
